csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -188,56 +193,35 @@
           Visible="false"
     />
     <None Include="$(PackageId).targets" 
-          PackagePath="\build\$(TargetFramework);\build\native"
-          Pack="true"
-          Visible="false"
-    />
-    <None Include="$(PackageId).props"
-          PackagePath="\build\$(TargetFramework);\build\native"
+          PackagePath="\build\$(TargetFramework)"
           Pack="true"
           Visible="false"
     />
+
     <!-- Some tools to be packaged in nightly build only, should not be released -->
     <!-- These are copied to the runtimes folder for convenience of loading with the dlls -->
     <None Include="$(NativeBuildOutputDir)\onnxruntime_perf_test.exe"
           Condition="('$(IsReleaseBuild)' != 'true') And ($(TargetArchitecture)=='x64') And Exists('$(NativeBuildOutputDir)\onnxruntime_perf_test.exe')"
           PackagePath="\runtimes\win-$(TargetArchitecture)\native"
-          Pack="true"
+          Pack="false"
```

Comment:
How does the condition interacts with Pack value?

Reply:
The condition doesn't. We want to prevent packing (hence set Pack = false) for the native dlls here as this packs all the managed stuff only.